sormas-backend/src/main/java/de/symeda/sormas/backend/caze/CaseFacadeEjb.java

```diff
@@ -1388,6 +1389,36 @@ public List<CaseSelectionDto> getSimilarCases(CaseSimilarityCriteria criteria) {
 		return entries;
 	}
 
+	@Override
+	public List<CaseDataDto> getRelevantCasesForVaccination(VaccinationDto vaccinationDto) {
```

Comment:
The relevance in terms of dates needs to be added to this method so that only relevant cases are returned, i.e. the resulting case list needs to be filtered accordingly.

Reply:
I've filtered the returned list from this methods in VaccinationController ( as you've also suggested in one of your comments)